build/bin/release

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+token=${GITHUB_TOKEN}
+branch=$(git branch --show-current)
+
+# Package the Helm chart
+helm package $chart --destination build/release
+
+# Upload the Helm chart release
+cr upload \
+    --git-base-url https://api.github.com/ \
+    --git-upload-url https://uploads.github.com/ \
+    --owner $owner \
+    --git-repo $repo \
+    --package-path build/release \
+    --token $token
+
+# Switch to the gh-pages branch
+git checkout gh-pages
+
+# Update the repository index
+cr index \
```

Comment:
Will this fail if trying to add chart with a version that has already been added to the index?

Reply:
I'm not sure there's a way to ensure that. The index updating logic doesn't seem to mind updating releases that are already present in the index. What prevents it is using GitHub releases as the source of truth for updates to `index.yaml`. As long as the releases haven't changed `index.yaml` should not change.

Reply:
This sounds good - the main concern was breaking repeatability by a user on the client side by re-releasing a changed version.